source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -115,48 +123,95 @@ Table 3.p (below): List of Data_Type Values
      - Repository\ |br|\ |lb|
        Scholarly_Collaboration_Network
-     - PR, IR\ |br|\ |lb|
-       PR_P1
+     - PR, IR
 
-   * - Database
-     - A fixed database where content is searched and accessed in the context of the database. A given item on the host may be in multiple databases but a transaction must be attributed to a specific database. Note that Data_Type Database is only applicable for Searches and Access Denied at the database level and for Investigations and Requests for Full_Content_Databases*.
+   * - Conference
+     - A collection of papers, posters, or recordings of material associated with a conference. Typically part of a serial publication.
      - A&I_Database\ |br|\ |lb|
        Aggregated_Full_Content\ |br|\ |lb|
        Discovery_Service\ |br|\ |lb|
+       eBook\ |br|\ |lb|
        eBook_Collection\ |br|\ |lb|
-       Full_Content_Database\ |br|\ |lb|
-       Multimedia_Collection
-     - PR, DR\ |br|\ |lb|
-       PR_P1, DR_D1, DR_D2
+       eJournal\ |br|\ |lb|
+       Repository\ |br|\ |lb|
+       Scholarly_Collaboration_Network
+     - PR, DR, TR
+
+   * - Conference_Item
+     - A single paper, poster, or recording of material associated with a conference. 
+     - Repository\ |br|\ |lb|
+       Scholarly_Collaboration_Network
+     - PR, IR
+
+   * - Database_Aggregated
+     - *Only applies to Denial and Search metrics.*
+     
+       Activity within an aggregated database of full text serial and/or monograph content, or content otherwise aggregated into titles. A given item on the host may be in multiple databases but a transaction must be attributed to a specific database. Activity that would result in Investigation and Request metrics must be reported against the appropriate title level Data_Type (e.g. Journal for a journal article).
+     - Aggregated_Full_Content
+     - DR
+
+   * - Database_AI
+     - *Only applies to Denial and Search metrics.*
+     
+       Activity within a fixed database where bibliographic metadata is searched and accessed in the context of the database. A given item on the host may be in multiple databases but a transaction must be attributed to a specific database. Activity that would result in Investigation and Request metrics must be reported against the appropriate Data_Type (e.g. Journal for a journal article).
+     - A&I_Database
+     - DR
+
+   * - Database_Full
+     - *Only applies to Denial and Search metrics.*
+     
+       Activity within databases that are a collection of content items that are not aggregated into titles. A given item on the host may be in multiple databases but a transaction must be attributed to a specific database. Activity that would result in Investigation and Request metrics must be reported against the appropriate item-level Data_Type (e.g. Multimedia).
+     - Full_Content_Database
```

Comment:
All Host_Types required to provide DR must be able to report Searches and Denials, so all these Host_Types must be listed under one of the database Data_Types, probably eBook_Collection under Database_Aggregated and Multimedia_Collection under Database_Full. I'm not sure about Discovery_Service, maybe Database_AI?